internal/gm-unit/editor_skills.lua

```diff
@@ -24,7 +24,13 @@ function list_skills(unit, learned_only)
                     u_skill={rating=-1,experience=0}
                 end
                 local rating
-                if u_skill.rating >=0 then
+                if u_skill.rating >15 then
```

Comment:
Please use `df.skill_rating.Legendary` instead of the magic number `15`

Reply:
I've replaced the magic number and added an entry in `changelog.txt`.
I'm unsure if I made the changelog entry correctly though.